introduction/java_output_formatting/Solution.java

```diff
@@ -0,0 +1,39 @@
+import java.util.Scanner;
+
+public class Solution {
+  static private final int REQUIRED_LENGTH = 15;
+  static private final int MAX_ONE_DIGIT_NUMBER = 9; 
+  static private final int MAX_TWO_DIGITS_NUMBER = 99;
+  static private final String ONE_DIGIT_NUMBER_PREFIX = "00";
+  static private final String TWO_DIGITS_NUMBER_PREFIX = "0";
+  
+  public static String formatOutput(String line, String numberPrefix, int number) {
+    String result = line;
+    for (int j = 0; j < REQUIRED_LENGTH - line.length(); j++) {
+      result += " ";
+    }
+    result += numberPrefix;
+    result += Integer.toString(number);
+    result += "\n";
+    return result;
+  }
+
+  public static void main(String[] args) {
+    Scanner scan = new Scanner(System.in);
+    System.out.println("================================");
```

Comment:
`String verticalSeparator = new String(new char[25]).replace('\0', '=');`
replace 25 with the correct number